static/docs/command-reference/import.md

```diff
@@ -1,10 +1,11 @@
 # import
 
-Download or copy an <abbr>output</abbr> file or directory from any <abbr>DVC
-repository</abbr> (e.g. hosted on GitHub) into the <abbr>workspace</abbr>. DVC
-creates a [DVC-file](/doc/user-guide/dvc-file-format) with information about the
-data source, which can later be used to [update](/doc/command-reference/update)
-the import.
+Download or copy a file or directory, possibly being an <abbr>output</abbr>,
```

Comment:
```suggestion
Download a file or directory, possibly being an <abbr>output</abbr>,
```
Needs rewrapping of course.

Reply:
Oops sorry I also meant to suggest that we can just remove the `, possibly being an <abbr>output</abbr>,` part from here. We want to keep the first paragraph as simple as possible.

Reply:
Having `<abbr>output</abbr>` somewhere in the description could be useful though, but would also need to add in the `dvc get` cmd ref... Up to you, not super needed.

Reply:
You know what, I'll take care of this post-merge. Thanks @Baranowski!

Reply:
Addressed in d99cb390.

Reply:
Thank you!